capsules/segmenter_person_openvino/backend.py

```diff
@@ -0,0 +1,162 @@
+from typing import Dict, List
+
+import numpy as np
+import cv2
+
+from vcap import (
+    DetectionNode,
+    Resize,
+    DETECTION_NODE_TYPE,
+    OPTION_TYPE,
+    BaseStreamState,
+    rect_to_coords)
+from vcap_utils import BaseOpenVINOBackend
+
+
+class Backend(BaseOpenVINOBackend):
+    label_map = {
+        1: "person"
+    }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        input_blob_name = self.input_blob_names[0]
+        input_blob = self.net.inputs[input_blob_name]
+        _, _, h, w = input_blob.shape
+
+        self.blob_h = h
+        self.blob_w = w
+
+    def process_frame(self, frame: np.ndarray,
+                      detection_node: DETECTION_NODE_TYPE,
+                      options: Dict[str, OPTION_TYPE],
+                      state: BaseStreamState) -> DETECTION_NODE_TYPE:
+        frame_h, frame_w, _ = frame.shape
+
+        if options["keep_aspect_ratio"]:
```

Comment:
Do you think people will want to turn this off? In other capsules, we always keep the same aspect ratio, right?